### DISCUSSION TITLE
Valores após a compilação mudam.

### DISCUSSION THREAD

**Discussion by broeringvictor** [balta-io/forum#689] [Tira Dúvidas] [answered]
Bom dia, professores! 

Fundamentos do C#
[2801] -  Interpolação de strings

## Após fazer a compilação e rodar no terminal, os valores do double são convertidos para o padrão brasileiro. "," ao em vez de ser ".", como que eu faço para deixar igual da Aula?

![image](https://github.com/user-attachments/assets/794f6ea9-6196-4334-a6ee-dbf26455a41d)

## A estrutura do .NET mudou da criação do curso para o .NET 8?

![image](https://github.com/user-attachments/assets/bb4e2efb-71ec-4b60-a4f2-7a74ec0d7243)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @broeringvictor isso ocorre pois sua máquina deve estar no idioma PT-BR, enquanto a minha está em inglês.
Não há necessidade de converter no momento, você vera mais sobre localização nos próximos cursos, fica tranquilo!
